00-Raster-Structure.Rmd

```diff
@@ -511,21 +529,19 @@ GDALinfo("NEON_RemoteSensing/HARV/DSM/HARV_dsmCrop.tif")
 
 Notice a few things in the output:
 
-1. A projection is described as a string - this format is called `proj4`
+1. A projection is described in a string - this format is called `proj4`:
```

Comment:
By making this change I worry that we loose the idea that THIS format is proj4 but other formats do exist.   Thoughts?